frontend/packages/console-shared/src/components/editor/YAMLEditorToolbar.tsx

```diff
@@ -1,17 +1,29 @@
 import * as React from 'react';
 import ShortcutsLink from './ShortcutsLink';
 import './YAMLEditorToolbar.scss';
+import { useTranslation } from 'react-i18next';
 
 interface YAMLEditorToolbarProps {
   showShortcuts?: boolean;
   toolbarLinks?: React.ReactNodeArray;
 }
 
 const YAMLEditorToolbar: React.FC<YAMLEditorToolbarProps> = ({ showShortcuts, toolbarLinks }) => {
+  const { t } = useTranslation();
   if (!showShortcuts && !toolbarLinks?.length) return null;
-
+  const isMac = window.navigator.platform.includes('Mac');
```

Comment:
Can you export the mac check from `Shortcut` so that the same logic is reused from a single area.
`export const isMac = () => window.navigator.platform.includes('Mac')`

Reply:
done